src/Classes/MyRadio/MyRadioLDAPAuthenticator.php

```diff
@@ -37,6 +37,11 @@ public function __destruct()
      */
     public function validateCredentials($user, $password)
     {
+        # Check that it looks like a legit user first.
+        if (!empty(Config::$auth_ldap_regex) && !preg_match(Config::$auth_ldap_regex, $user)) {
+            sleep(0.01); # Emulate the speed of LDAP to help with timing attacks.
```

Comment:
Not sure this really helps with timing attacks. At least needs to be an random amount of sleeping, but I'd say the whole thing is necessary anyway

Reply:
Cheers :)